.travis.yml

```diff
@@ -6,11 +6,9 @@ install:
   - cp config/database.yml.example config/database.yml
   - cp config/config.yml.example config/config.yml
   - cp db/schema.rb.example db/schema.rb
-  - docker-compose build
-  - docker-compose run web sleep 10
-  - docker-compose run web bash -c "rake db:setup"
-  - docker-compose run web bash -c "rake db:migrate"
-  - docker-compose run web bower install --allow-root
+  - docker-compose up -d --build
+  - docker-compose exec web bundle install
+  - docker-compose exec web /bin/bash -lc "rake db:setup || rake db:migrate"
```

Comment:
If it has new migrations it'll execute, if not it won't :D

Reply:
Clever!